python/src/python_interface.cpp

```diff
@@ -85,8 +91,12 @@ std::pair< py::list, py::list > impedance( const py::list& input,
 
         py::array_t< float > numpy_array = py::cast< py::array >( item );
         if ( numpy_array.strides( 2 ) != sizeof( float ) ) {
-            throw std::runtime_error(
-                "The NumPy arrays must be contiguous in trace direction." );
+            const std::string msg{
+                "The NumPy arrays must have stride " +
+                std::to_string( sizeof( float ) ) +
+                "in trace dimension to be contiguous in this dimension. " +
+                "Got stride" + std::to_string( numpy_array.strides( 2 ) ) + "." };
```

Comment:
Got stride **space** :smiling_imp: 

And probably line above before "in trace dimension" too

Reply:
🙈